app.rb

```diff
@@ -13,8 +13,13 @@ class Uploader < Sinatra::Base
 
     get '/healthcheck' do
       {
+        service_status: 'OK',
```

Comment:
I'm assuming the "OK" here will eventually be computed from the values of the dependent checks - i.e. return 'OK' only if all the dependent statuses are good.
Even if that's the eventual plan, I think it would be good to build that logic in now, so we can add to it, so that it doesn't get forgotten about, later.